File: src/test/java/com/polykhel/billnext/service/mapper/WalletGroupMapperTest.java
package com.polykhel.billnext.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class WalletGroupMapperTest {

    private WalletGroupMapper walletGroupMapper;

    @BeforeEach
    public void setUp() {
        walletGroupMapper = new WalletGroupMapperImpl();
    }
}

File: src/test/kotlin/com/polykhel/billnext/service/mapper/WalletGroupMapperTest.kt
package com.polykhel.billnext.service.mapper

import org.junit.jupiter.api.BeforeEach

class WalletGroupMapperTest {

    private lateinit var walletGroupMapper: WalletGroupMapper

    @BeforeEach
    fun setUp() {
        walletGroupMapper = WalletGroupMapperImpl()
    }
}
